linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminFilter.scala

```diff
@@ -14,8 +14,9 @@ class AdminFilter(adminHandler: AdminHandler, css: Seq[String] = Nil) extends Si
     service: Service[Request, Response]
   ): Future[Response] = {
     service(request).map { rsp =>
+      val itemToHighlight = request.path.replace("/", "")
```

Comment:
great question. it does! I tested it out with admin.yaml and it works. I asked @siggy and he says the admin scala is unaware of the root url.

if you use admin.yaml that serves admin at /foo/ and inspect request.path, it's unaware that it's at /foo/... so request.path here is still just "/requests", "/namerd" etc...